src/app/elastic/search.py

```diff
@@ -39,8 +40,37 @@ def node_filter(self, resource_type: ResourceType, node_id: uuid.UUID) -> Search
         search = self.type_filter(resource_type=resource_type)
 
         if resource_type is ResourceType.COLLECTION:
-            return search.filter(qterm(qfield=ElasticResourceAttribute.PATH, value=node_id))
-        return search.filter(qterm(qfield=ElasticResourceAttribute.COLLECTION_PATH, value=node_id))
+            return search.filter(
+                qterm(qfield=ElasticResourceAttribute.PATH, value=node_id)
+            )
+        return search.filter(
+            qterm(qfield=ElasticResourceAttribute.COLLECTION_PATH, value=node_id)
+        )
+
+    def filter_serial_objects_out(self):
```

Comment:
I assume if it has the childobject attribute it means it is a "Serienobjekt"?
Can we find a better name for this? - I cant really come up with any...
But i think it should be series instead of serial :)
Maybe:
 - filter_non_series_objects()
 - drop_series_objects()

Reply:
non_series_filter() to be in line with the other ..._filter

Reply:
Refactored to non_series_objects_filter